apps/ledger-live-mobile/src/screens/Portfolio/index.tsx

```diff
@@ -58,9 +65,41 @@ function PortfolioScreen({ navigation }: NavigationProps) {
   const { t } = useTranslation();
 
   const discreetMode = useSelector(discreetModeSelector);
+  const hasBeenUpsoldProtect = useSelector(hasBeenUpsoldProtectSelector);
+  const lastConnectedDevice = useSelector(lastConnectedDeviceSelector);
   const [isAddModalOpened, setAddModalOpened] = useState(false);
   const { colors } = useTheme();
   const { isAWalletCardDisplayed } = useDynamicContent();
+  const protectFeature = useFeature("protectServicesMobile");
+  const postOnboardingURL =
+    protectFeature?.params?.onboardingRestore?.postOnboardingURI;
+  const dispatch = useDispatch();
+
+  const internetReachable = async () => {
+    // workaround : on iOS, netInfo returns null on the first call, see ; https://github.com/react-native-netinfo/react-native-netinfo/issues/572
+    let isInternetReachable = false;
+
+    isInternetReachable = !!(await NetInfo.fetch()).isInternetReachable;
+
+    if (isInternetReachable) {
+      return isInternetReachable;
+    }
```

Comment:
if it returns false you still try it again, maybe just do that: if `result===null` retry, else return `result`